Makefile

```diff
@@ -5,101 +5,91 @@ SHELL= /bin/bash
 TAG ?= 0.0.2
 PKG = github.com/integr8ly/gitea-operator
 COMPILE_OUTPUT = build/_output/bin/gitea-operator
-.PHONY: check-gofmt
-check-gofmt:
-	diff -u <(echo -n) <(gofmt -d `find . -type f -name '*.go' -not -path "./vendor/*"`)
 
-.PHONY: test-unit
-test-unit:
-	@echo Running tests:
-	go test -v -race -cover ./pkg/...
-
-.PHONY: test-e2e-local
-test-e2e-local:
-	@echo Running e2e tests:
-	operator-sdk test local ./test/e2e --go-test-flags "-v"
-
-.PHONY: setup
-setup:
+.PHONY: setup/dep
```

Comment:
should we be running `dep ensure` in this step?

Reply:
We are checking in vendor files so I don't think it is necessary

Reply:
I tried `make code/compile` after switching to this branch and it failed with an error. Running `dep ensure` fixed it.